package.json

```diff
@@ -24,11 +24,21 @@
   ],
   "scripts": {
     "test": "wct",
+    "check": "npm-run-all --parallel check:*",
+    "check:bower": "magi check-bower",
+    "check:version": "magi check-version",
+    "lint": "npm-run-all --parallel lint:*",
```

Comment:
How does this show the output from the parallel running tasks? Just wondering if it can be a problem if e.g. more than one lint task outputs multi-line info at the same time. Can the output of different commands be interleaved together so that it's unreadable?

Reply:
See docs here: https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#run-scripts-in-parallel

> If a script exited with a non-zero code, the other scripts and those descendant processes are killed